indico/MaKaC/review.py

```diff
@@ -1121,6 +699,7 @@ def newAbstract(self, av, **data):
         id = self._generateNewAbstractId()
         a = Abstract(self, id, av, **data)
         self._abstracts[id] = a
+        self._new_abstract(a, data)
```

Comment:
I'd get rid of `_generateNewAbstractId` and let postgres generate the id (last_friendly_abstract_id on the event)